source/features/more-dropdown.tsx

```diff
@@ -13,15 +13,15 @@ const repoUrl = getRepoURL();
 function createDropdown(): void {
 	// Markup copied from native GHE dropdown
 	appendBefore('.reponav', '[href$="settings"]',
-		<div className="reponav-dropdown js-menu-container">
-			<button type="button" className="btn-link reponav-item js-menu-target" aria-expanded="false" aria-haspopup="true">
-				{'More '}
-				<span className="dropdown-caret"/>
-			</button>
-			<div className="dropdown-menu-content js-menu-content">
-				<div className="dropdown-menu dropdown-menu-se"/>
-			</div>
-		</div>
+		<details className="reponav-dropdown details-overlay details-reset">
+			<summary>
+				<span className="btn-link reponav-item">
```

Comment:
I think a `role="button"` wouldn't hurt here — any a11y experts out there to correct me if I'm wrong on this?

Reply:
I don't quite know about how to properly apply a11y here, but GitHub will add `aria-haspopup="menu"` to `<summary>` and other dropdowns don't have extra attributes so I guess it's fine.